relayer-cli/src/commands/query/client.rs

```diff
@@ -289,8 +289,11 @@ impl Runnable for QueryClientConnectionsCmd {
         status_info!("Options", "{:?}", opts);
 
         let chain = CosmosSDKChain::from_config(chain_config).unwrap();
-        let res =
-            chain.query::<Vec<String>>(ClientConnections(opts.client_id), opts.height, opts.proof);
+        let res = chain.query::<Vec<ConnectionId>>(
+            ClientConnections(opts.client_id),
+            opts.height,
+            opts.proof,
```

Comment:
Should remove `opts.proof`, this is querying the private store (see https://github.com/cosmos/ics/tree/master/spec/ics-024-host-requirements).
Have you tried to query with proof? What is the result?

Reply:
if you specify `-p true` you get a proper error saying it's a privateStore `client connections query error store error: requested proof for privateStore path` so I believe this is OK. I could hard code the value in the function call instead of using `opts.proof` as argument and set it to `false` but I think the way it is probably better.

Reply:
From an UX experience I find it strange that we offer an option that we know it is not valid. My preference is to just remove it all together and hardcode `false`. Or at least change the default to `false`. I realize the client state falls under the same debate. Maybe the best thing is to open a separate issue and discuss there with the whole team.

Reply:
Open a separate issue to handle this discussion #205